src/view/com/modals/AddAppPasswords.tsx

```diff
@@ -119,15 +122,19 @@ export function Component({}: {}) {
       <View>
         {!appPassword ? (
           <Text type="lg" style={[pal.text]}>
-            Please enter a unique name for this App Password or use our randomly
-            generated one.
+            <Trans>
+              Please enter a unique name for this App Password or use our
+              randomly generated one.
+            </Trans>
           </Text>
         ) : (
           <Text type="lg" style={[pal.text]}>
-            <Text type="lg-bold" style={[pal.text]}>
-              Here is your app password.
-            </Text>{' '}
-            Use this to sign into the other app along with your handle.
+            <Trans>
+              <Text type="lg-bold" style={[pal.text]}>
+                Here is your app password.
+              </Text>{' '}
```

Comment:
Does this work?

Reply:
Yes! Lingui has support for nested components since `<Trans>` is a macro

Reply:
Nice nice